api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -116,7 +116,7 @@ type VRGSyncSpec struct {
 type VolumeReplicationGroupSpec struct {
 	// Label selector to identify all the PVCs that are in this group
 	// that needs to be replicated to the peer cluster.
-	PVCSelector metav1.LabelSelector `json:"pvcSelector"`
+	PVCSelector metav1.LabelSelector `json:"pvcSelector,omitempty"`
```

Comment:
I believe this should be a required field.  Without it, Ramen will not know what to protect.